scripts/prepare

```diff
@@ -33,10 +33,28 @@ for f in packages/*; do
 				basename=$(basename -- ${file})
 				patch -p3 -d ${f}/charts < ${f}/$basename
 			done
+			copied_dependencies=()
+			if [[ -f ${f}/charts/requirements.yaml ]]; then
+				repos=$(cat ${f}/charts/requirements.yaml | yq r - "dependencies.*.repository")
+				while IFS= read -r repo; do
+					if [[ ${repo} == file://* ]]; then
+						charts_path=${repo/file:\/\//${f}/charts/}
+						overlay_path=${repo/file:\/\//${f}/overlay/}
+						if [[ ! -d $charts_path ]] && [[ -d $overlay_path ]]; then
+							cp -R $overlay_path $charts_path
+							copied_dependencies+=("${charts_path}")
+						fi
+					fi
+				done < <(echo "${repos}")
+
+			fi
 			pwd=$(pwd)
 			cd ${f}/charts
 			helm dependency update
```

Comment:
Will `helm dependency update` generate dependency tarball files if you copied overlay file into chart directory?

Reply:
Yes, this allows developers to locally test their charts with `helm install <release> ./packages/<pkg>/charts` before running the `make charts` step; however, they will be ignored in the patch step due to a change added in https://github.com/rancher/dev-charts/pull/28 that ignores `.tgz` files.